### DISCUSSION TITLE
Methane jet cannot be ignited at relatively high exit velocity (30-60 m/s)?

### DISCUSSION THREAD

**Discussion by lindaxia** [firemodels/fds#13356] [Q&A]
Hi all,
I would like to study the combustion/vortex development characteristics of a methane jet flame at relatively high exit velocity (30-60 m/s). In my experimental research, we observed the lift-off phenomenon, which means the flame doesn't attach to the fuel exit and pulsation at downstream. However, the lift-off phenomenon was not detected when using the default reaction (&REAC FUEL = 'METHANE'), which is hypothesized to be due to the default mechanism of mixing-controlled combustion model. Therefore, the &REAC of methane is reset using the guidelines for finite rate combustion (chapter 13.3 of the user's guide). The parameters A,E, etc. are based on the study in [1].
Here is the problem: The fuel cannot be ignited after changing &REAC setting. We tried setting several high-temperature ignition sources and increasing the ignition time did not solve this problem. Additionally, Propane (example in Chapter 13.3) cannot be ignited ether. I'm not sure why this happens. I hope you will not hesitate to advise. Thank you all.

[1] C.K. Westbrook, F.L.J.C.s. Dryer, technology, Simplified reaction mechanisms for the oxidation of hydrocarbon fuels in flames, 27 (1981) 31-43.

The code is attached below.
&HEAD CHID = 'verticaljetfire_model07'
      TITLE = 'verticaljetfire_model07'   /
   
&MESH ID = 'MESH01'
      XB = 0 0.5 0 0.5 0 0.2
      IJK = 50 50 20
      MPI_PROCESS = 0 /
&MESH ID = 'MESH02'
      XB = 0 0.5 0 0.5 0.2 0.4
      IJK = 50 50 20
      MPI_PROCESS = 1 /
&MESH ID = 'MESH03'
      XB = 0 0.5 0 0.5 0.4 0.6
      IJK = 50 50 20
      MPI_PROCESS = 2 / 
&MESH ID = 'MESH04'
      XB = 0 0.5 0 0.5 0.6 0.8
      IJK = 50 50 20
      MPI_PROCESS = 3 /
&MESH ID = 'MESH05'
      XB = 0 0.5 0 0.5 0.8 1
      IJK = 50 50 20
      MPI_PROCESS = 4 /

&MISC TMPA = 10
      RESTART = .False.
      SIMULATION_MODE = 'DNS' /
&DUMP NFRAMES = 2000
      DT_RESTART = 1 /
&TIME T_END = 10 /                                            
      
&VENT MB = XMIN          SURF_ID = 'OPEN' /
&VENT MB = XMAX          SURF_ID = 'OPEN' /
&VENT MB = YMIN          SURF_ID = 'OPEN' /
&VENT MB = YMAX          SURF_ID = 'OPEN' /
&VENT MB = ZMAX          SURF_ID = 'OPEN' /

&SPEC ID='NITROGEN'/
&SPEC ID='METHANE' /
&SPEC ID='OXYGEN', BACKGROUND=T /
&SPEC ID='WATER VAPOR' /
&SPEC ID='CARBON DIOXIDE' /

&REAC ID = 'R1'
      FUEL = 'METHANE'
      A = 1.3e8
      E = 202457
      SPEC_ID_NU = 'METHANE','OXYGEN','CARBON DIOXIDE','WATER VAPOR'
      NU = -1,-2,1,2
      SPEC_ID_N_S = 'METHANE','OXYGEN'
      N_S = -0.3,1.3 /

&SURF ID = 'BURNER'
      SPEC_ID = 'METHANE'   
      MASS_FRACTION = 1
      VEL = -10
      RGB = 255 0 0 /
&OBST XB =  0.24 0.255 0.24 0.255 0 0.075
      SURF_ID = 'INERT'
      RGB = 128 64 64 /
&VENT XB = 0.24 0.255 0.24 0.255 0.075 0.075
      SURF_ID = 'BURNER' /

&SLCF PBY = 0.2475
      QUANTITY = 'TEMPERATURE'
      VECTOR = T /
&SLCF PBY = 0.2475
      QUANTITY = 'VELOCITY'
      VECTOR = T /
&SLCF PBZ = 0.175
      QUANTITY = 'TEMPERATURE'
      VECTOR = T /
&SLCF PBZ = 0.175
      QUANTITY = 'VELOCITY'
      VECTOR = T /
&SLCF PBZ = 0.275
      QUANTITY = 'TEMPERATURE'
      VECTOR = T /
&SLCF PBZ = 0.275
      QUANTITY = 'VELOCITY'
      VECTOR = T /
&SLCF PBZ = 0.475
      QUANTITY = 'TEMPERATURE'
      VECTOR = T /
&SLCF PBZ = 0.475
      QUANTITY = 'VELOCITY'
      VECTOR = T /
&SLCF PBZ = 0.775
      QUANTITY = 'TEMPERATURE'
      VECTOR = T /
&SLCF PBZ = 0.775
      QUANTITY = 'VELOCITY'
      VECTOR = T /
      
&DEVC ID = 'TIMER'     XYZ = 0.255 0.255 0.095     SETPOINT = 1   QUANTITY = 'TIME'   INITIAL_STATE = T /
&SURF ID = 'IGNITER'
      TMP_FRONT = 3000
      COLOR = 'GREEN' /
&OBST XB = 0.255 0.260 0.245 0.250 0.09 0.195
      DEVC_ID = 'TIMER'
      SURF_ID6 = 'IGNITER','INERT','INERT','INERT','IGNITER','INERT' /

[Methane_jet_flame.txt](https://github.com/user-attachments/files/16742745/Methane_jet_flame.txt)

**Comment by rmcdermo** [comment]
I suspect that temperature resolution is the main problem.  But you must realize you are in research territory with FDS.

I would first focus on premixed flame speed calculations.  If you cannot match flame speed, I do not see how you will match lift off height.  If you plan to do this with LES, that is even more challenging.

**Reply by lindaxia** [reply]
Thank you for your quick reply. Based on the calculation results, I think the velocity distribution was correct. I noticed that you mentioned LES, which is exactly what I wanted to ask. I've found that there is a large discrepancy between the calculation results using the DNS settings (mainly the speed results, due to unignited) and the LES calculation results. Based on my experimental result, I think the numerical velocity results of the LES are closer to my understanding. I don't really understand why that would lead to such a discrepancy. I wonder if you can answer my confusion? 
Also, I'd like to know more about what you see as my difficulty in using LES for this task?

**Reply by rmcdermo** [reply]
You are jumping too far ahead.  Lift off needs a precise balance between flame speed and velocity of the jet.  Flame speed depends strongly on local temperature, transport properties, curvature of the flow field, etc.  Finite rate chemistry with LES in FDS has NEVER been done.  The primary issue is temperature of the flame region and the fact that you have no representation of a premixed flame in FDS (I hate to point you toward INITIAL_UNMIXED_FRACTION, but this is the what you'd have to look at if you continue down this road).  I have to believe there are other codes that are better suited for modeling jet flames.
